app/controllers/admin_application_controller.rb

```diff
@@ -0,0 +1,4 @@
+class AdminApplicationController < ActionController::Base
+  before_action :authenticate_admin!
+  render 'layouts/admin.html.erb'
```

Comment:
setting up a layout in the controller should be done in another way
https://api.rubyonrails.org/classes/ActionView/Layouts.html